about/pricing.html.md.erb

```diff
@@ -64,7 +64,9 @@ Volume billing is pro-rated to the hour.
 
 ### Anycast IP addresses
 
-Each application includes one Anycast IPv4 and unlimited Anycast IPv6 Anycast addresses for global load balancing. Additional IPv4 addresses are $2/mo.
+Each application receives a [shared IPv4 address](/docs/reference/services/#shared-ipv4) and unlimited [Anycast IPv6](/docs/reference/services/#ipv6) addresses for global load balancing.
+
+Dedicated IPv4 addresses are $2/mo.
```

Comment:
Not sure if we know the answer, but I'd like to see something akin to `Volume billing is pro-rated to the hour.` here, just to be clear what's being charged there.

Reply:
Do we know that we're going to pro-rate them?

Reply:
ah ok-- if they're not, I'd rather read `ips are not pro-rated` etc..